src/main/java/it/gov/pagopa/pu/debtpositions/controller/DebtPositionControllerImpl.java

```diff
@@ -96,11 +102,17 @@ public ResponseEntity<Void> installmentSynchronize(DebtPositionOrigin origin, In
       .executionConfig(installmentSynchronizeDTO.getExecutionConfig())
       .build();
     WorkflowCreatedDTO workflow = installmentSynchronizeService.installmentSynchronize(installmentSynchronizeDTO, wfExecutionParameters, origin, accessToken, operatorExternalUserId);
-    return ResponseEntity
-      .status(HttpStatus.CREATED)
-      .header(HEADER_X_WORKFLOW_ID, workflow.getWorkflowId())
-      .header(HEADER_X_RUN_ID, workflow.getRunId())
-      .build();
+    if(workflow != null) {
+      return ResponseEntity
+        .status(HttpStatus.CREATED)
+        .header(HEADER_X_WORKFLOW_ID, workflow.getWorkflowId())
+        .header(HEADER_X_RUN_ID, workflow.getRunId())
+        .build();
+    } else {
+      return ResponseEntity
+        .status(HttpStatus.NO_CONTENT)
```

Comment:
I would still return OK, but also for the workflow != null useCase...
even if it has not created a WF (which is a secundary effect), it will always operate with db data